facebook-commerce.php

```diff
@@ -1398,6 +1416,41 @@ function update_product_group( $woo_product ) {
 		*/
 	}
 
+
+	/**
+	 * Determine if there matching variation to the default attributes
+	 *
+	 * @param WC_Facebook_Product $woo_product
+	 *
+	 * @return WC_Product_Variation|null
+	 */
+	protected function get_product_group_default_variation( $woo_product ) {
```

Comment:
Maybe I just used to make methods like that `protected` by default to avoid changing it down the road in case the the class got extended ... I just need to change this habit